lib/glific/notion.ex

```diff
@@ -0,0 +1,292 @@
+defmodule Glific.Notion do
+  @moduledoc """
+  Notion API integration Utilities
+  """
+  alias Glific.Registrations.Registration
+
+  @doc """
+  Utilities to interact with Notion
+  """
+
+  @ngo_database_id "880cf3440d4c4e3f80453103e712f897"
```

Comment:
do we need to make this public? I know if keys are invalid then API will fail, but just double check it